src/openrct2/paint/tile_element/Paint.LargeScenery.cpp

```diff
@@ -139,7 +138,8 @@ static void large_scenery_sign_paint_line(
         acc -= (width / 2);
     }
     uint32_t codepoint;
-    while ((codepoint = utf8_get_next(fitStr, &fitStr)) != 0)
+    const utf8* fitStrPtr = fitStr;
+    while ((codepoint = utf8_get_next(fitStr, &fitStrPtr)) != 0)
```

Comment:
I'm getting a hang here because fitStr is never updated to the next character.

Reply:
How did you reproduce it?

Reply:
Couldn't even load the title screen

Reply:
Fixed